src/parseTools.js

```diff
@@ -596,14 +596,18 @@ function makeReturn64(value) {
   return `(setTempRet0(${pair[1]}), ${pair[0]})`;
 }
 
-function makeThrow(what) {
+function makeThrow(excPtr) {
   if (ASSERTIONS && DISABLE_EXCEPTION_CATCHING) {
-    what += ' + " - Exception catching is disabled, this exception cannot be caught. Compile with -sNO_DISABLE_EXCEPTION_CATCHING or -sEXCEPTION_CATCHING_ALLOWED=[..] to catch."';
+    var assertInfo = 'Exception catching is disabled, this exception cannot be caught. Compile with -sNO_DISABLE_EXCEPTION_CATCHING or -sEXCEPTION_CATCHING_ALLOWED=[..] to catch.';
     if (MAIN_MODULE) {
-      what += ' + " (note: in dynamic linking, if a side module wants exceptions, the main module must be built with that support)"';
+      assertInfo += ' (note: in dynamic linking, if a side module wants exceptions, the main module must be built with that support)';
```

Comment:
So previously would include the exception pointer as part of this message but now we don't?

Reply:
Indeed, it seemed a bit odd to me that the (meaningless) exception pointer was included within the assert info. 

So, this changes this message:
```console
$ echo -e '#include <stdexcept>\nint main(){throw std::invalid_argument("Invalid argument!");}' | ./emcc -g -fexceptions -sDISABLE_EXCEPTION_CATCHING -xc++ -o test.js -
$ node --unhandled-rejections=warn test.js
(node:48779) UnhandledPromiseRejectionWarning: 66840 - Exception catching is disabled, this exception cannot be caught. Compile with -sNO_DISABLE_EXCEPTION_CATCHING or -sEXCEPTION_CATCHING_ALLOWED=[..] to catch.
...
```

To:
```console
$ echo -e '#include <stdexcept>\nint main(){throw std::invalid_argument("Invalid argument!");}' | ./emcc -g -fexceptions -sDISABLE_EXCEPTION_CATCHING -xc++ -o test.js -
$ node --unhandled-rejections=warn test.js
(node:49174) UnhandledPromiseRejectionWarning: Exception catching is disabled, this exception cannot be caught. Compile with -sNO_DISABLE_EXCEPTION_CATCHING or -sEXCEPTION_CATCHING_ALLOWED=[..] to catch.
...
```

Reply:
Maybe lets leave that as is, or make it a separate PR, since its seems like a somewhat orthogonal issue?

Reply:
I'll open another PR for this. Commit c34a85b0aebe3faa445703fe71c1da6d08b5b62d restores this to its original state.